pokemon-checker/src/DataTransferObjects/PokemonDTO.ts

```diff
@@ -30,7 +30,7 @@ export type pokemonDisplayObj = {
   };
 };
 
-export type pokemonAbilities = [ability: AbilityDTO, isHidden: boolean];
+export type pokemonAbilities = { ability: AbilityDTO; isHidden: boolean };
```

Comment:
This is the primary change.

Once a pokemon (and its abilities) are stored and fetched, it should be easier to understand the abilities they're holding.